### ISSUE TITLE
Error when fuzzing `Expected to find an object with property ['th'] in path $['th'] but found 'net.minidev.json.JSONArray'`

### ISSUE THREAD

**Issue by qarmin** [Endava/cats#119]
**Describe the bug**
When trying to run `./cats random --server="$IP_ADDRESS" --contract=$DOWNLOAD_PATH --mc 500 -X POST --stopAfterTimeInSec 10 --path /rest/1/pdf/` on one openapi, then I get instantly

```
com.jayway.jsonpath.PathNotFoundException: Expected to find an object with property ['th'] in path $['th'] but found 'net.minidev.json.JSONArray'. This is not a json object according to the JsonProvider: 'com.jayway.jsonpath.spi.json.JsonSmartJsonProvider'.
```
this works fine with openapi file with removed some properties
- problematic openapi - [broken_openapi.json](https://github.com/Endava/cats/files/14772513/broken_openapi.json)
- broken openapi - [good_openapi.json](https://github.com/Endava/cats/files/14772514/good_openapi.json)

**Environment:**
```
Key           | Value
------------- | --------------------
OS Name       | Linux
OS Version    | 6.5.0-26-generic
OS Arch       | amd64
Binary Type   | native
Cats Version  | 11.3.0
Cats Build    | 2024-03-15T07:10:32Z
Term Width    | 213
Term Type     | xterm-256color
Shell         | /bin/bash

```

**Additional context**

**Comment by en-milie** [comment]
It should be fixed in commit [f5ef319](https://github.com/Endava/cats/commit/f5ef31942784b06f606d315fd59312d29cbe2c9d)

**Comment by en-milie** [comment]
This is fixed in: https://github.com/Endava/cats/releases/tag/cats-11.4.0
